corehq/apps/data_interfaces/static/data_interfaces/js/auto_update_rules.js

```diff
@@ -98,10 +98,38 @@ hqDefine("data_interfaces/js/auto_update_rules", [
         return self;
     };
 
+    var RuleRunHistoryViewModel = function (ruleRuns) {
+        var self = {};
+        self.ruleRuns = ko.mapping.fromJS(ruleRuns);
+        self.paginatedRuleRuns = ko.observableArray([]);
+        // pagination
+        self.itemsPerPage = ko.observable(5);
```

Comment:
How was 5 chosen here?

Reply:
Good observation. Didn't think about this at all and it was picked because that's how I set it up on last page I did pagination on 🙃. Bumped it to 25 in 37615672e5f10f75540937d8db7b55b834d740d4. Let me know if you think that is adequate or should be higher.

Reply:
25 seems good to me. Thanks!